lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
Admittedly it's a bit odd that it just attempts to find a class from the supplied name, and forms are meant to follow a pattern but don't even have a common base. I'd be happy if we improved that so it's more scoped.